### ISSUE TITLE
[Usability]Search Results are difficult to navigate

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9608]
Copies from: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1853049/

Steps to reproduce:
Task 7: Search for packages

BL1 Details:
Device Type: Windows 11 Professional Version 22H2 (OS Build 22621.1555), Chrome Version 114.0.5735.110
Screen Reader: JAWS Professional Edition Version 2023.2212.23

BL2 Details:
Device Type: Windows 11 Home Version 22H2 (OS Build 22621.1848), Microsoft Edge version 116.0.1908.0
Screen Reader: NVDA version 2023.1

Issue Details:
Participant was uncertain where second result started. He stated that these results would be better in a tabular form.
Participant stated that each search result should be a heading, which would allow her to navigate to each result easily.

Recommendation:
Consider showing results in a table that would allow the advantages of table navigation for screen reader users.

**Comment by theot12** [comment]
Determine next steps.
